packages/design-system/src/components/popup/utils/getOffset.js

```diff
@@ -64,61 +79,45 @@ export function getYOffset(placement, spacing = 0, anchorRect) {
   }
 }
 
-/** @typedef {import('react').MutableRefObject} MutableRefObject */
-
-/**
- * @typedef {Object} Spacing Spacing
- * @property {number} x Horizontal spacing.
- * @property {number} y Vertical spacing.
- */
-/**
- * @typedef {Object} Offset Offset
- * @property {number} x Horizontal position.
- * @property {number} y Vertical position.
- * @property {number} width Width.
- * @property {number} height Height.
- */
-
-/**
- *
- * @param {string} placement Placement.
- * @param {Spacing} spacing Spacing.
- * @param {MutableRefObject<HTMLElement>} anchor Anchor element
- * @param {MutableRefObject<HTMLElement>} dock Dock element.
- * @param {MutableRefObject<HTMLElement>} popup Popup element.
- * @return {Offset} Popup offset.
- */
-export function getOffset(placement, spacing, anchor, dock, popup) {
+export function getOffset(placement, spacing, anchor, dock, popup, isRTL) {
```

Comment:
Why remove the JSDoc?

Reply:
Removed it by mistake added in https://github.com/google/web-stories-wp/pull/10197/commits/249932807171078b645d44e02514c784dc44a594